content/en/docs/reference/scheduling/config.md

```diff
@@ -255,10 +226,30 @@ the same configuration parameters (if applicable). This is because the scheduler
 only has one pending pods queue.
 {{< /note >}}
 
+## Migration introductions between api versions
```

Comment:
What does “introductions” mean here? I can't work it out.

Reply:
I just mean some explanations about the changes between different api versions.